docs/source/auth_rules.md

```diff
@@ -382,47 +382,42 @@
     <td><sub>TRUSTEE, STEWARD, NETWORK_MONITOR</sub></td>
     <td><sub>Getting validator_info from pool</sub></td>
   </tr>
-</table>
-
-### Also, there is a some optional rules for case if in config option ANYONE_CAN_WRITE is set to True:
-<table class="tg">
+  
   <tr>
-    <th>Transaction type</th>
-    <th>Field</th>
-    <th>Previous value</th>
-    <th>New value</th>
-    <th>Who can</th>
-    <th>Description</th>
+    <td><sub>REVOC_REG_DEF</sub></td>
+    <td><sub><code>*</code></sub></td>
+    <td><sub><code>*</code></sub></td>
+    <td><sub><code>*</code></sub></td>
+    <td><sub>TRUSTEE, STEWARD, TRUST_ANCHOR</sub></td>
+    <td><sub>Adding new REVOC_REG_DEF</sub></td>
   </tr>
   <tr>
-    <td><sub>NYM</sub></td>
-    <td><sub><code>role</code></sub></td>
-    <td><sub><code>&lt;empty&gt;</code></sub></td>
-    <td><sub><code>&lt;empty&gt;</code></sub></td>
-    <td><sub>Anyone</sub></td>
-    <td><sub>Adding new nym</sub></td>
+    <td><sub>REVOC_REG_DEF</sub></td>
+    <td><sub><code>*</code></sub></td>
+    <td><sub><code>*</code></sub></td>
+    <td><sub><code>*</code></sub></td>
+    <td><sub>Only owners can edit existing REVOC_REG_DEF</sub></td>
+    <td><sub>Editing REVOC_REG_DEF</sub></td>
   </tr>
   <tr>
-    <td><sub>SCHEMA</sub></td>
+    <td><sub>REVOC_REG_ENTRY</sub></td>
     <td><sub><code>*</code></sub></td>
     <td><sub><code>*</code></sub></td>
     <td><sub><code>*</code></sub></td>
-    <td><sub>Anyone</sub></td>
-    <td><sub>Any operations with SCHEMA transaction</sub></td>
+    <td><sub>Only the owner of the corresponding REVOC_REG_DEF can create new REVOC_REG_ENTRY</sub></td>
+    <td><sub>Adding new REVOC_REG_ENTRY</sub></td>
   </tr>
   <tr>
-    <td><sub>CLAIM_DEF</sub></td>
+    <td><sub>REVOC_REG_ENTRY</sub></td>
     <td><sub><code>*</code></sub></td>
     <td><sub><code>*</code></sub></td>
     <td><sub><code>*</code></sub></td>
-    <td><sub>Anyone</sub></td>
-    <td><sub>Any operations with CLAIM_DEF transaction</sub></td>
+    <td><sub>Only owners can edit existing REVOC_REG_ENTRY</sub></td>
+    <td><sub>Editing REVOC_REG_ENTRY</sub></td>
   </tr>
 </table>
 
-
-### As of now it's not implemented yet, but the next rules for Revocation feature are needed:
-#### If ANYONE_CAN_WRITE is set to False:
+### Also, there is a some optional rules for case if in config option ANYONE_CAN_WRITE is set to True:
```

Comment:
There are duplicate column headers in ANYONE_CAN_WRITE table: https://github.com/hyperledger/indy-node/blob/d6e66ca90775efaf7f69ea452c892dac2ee4cff1/docs/source/auth_rules.md